src/app/(proper_react)/redesign/(authenticated)/user/dashboard/DashboardTopBanner/DashboardTopBannerContent.tsx

```diff
@@ -473,9 +473,8 @@ export const DashboardTopBannerContent = (props: DashboardTopBannerProps) => {
                 "dashboard-top-banner-your-data-scan-in-progress-all-fixed-description",
                 {
                   starting_exposure_total_num:
-                    bannerData.totalExposures -
-                    bannerData.dataBrokerFixedExposuresNum -
-                    bannerData.dataBreachFixedExposuresNum -
+                    bannerData.dataBrokerFixedExposuresNum +
+                    bannerData.dataBreachFixedExposuresNum +
                     bannerData.dataBrokerInProgressExposuresNum,
```

Comment:
Yes! The ones “in progress” we are classifying as “fixed” on the dashboard.

Reply:
@flozia hmm so is this the number for "fixed" tab or "actions needed"
Because the var `starting_exposure_total_num` seems to suggest that it's unresolved

Reply:
This is the number for the “fixed” tab. Right, I should probably also change the variable name: It’s specifically for [this view](https://deploy-preview-3640--fx-monitor-storybook.netlify.app/?path=/story/pages-dashboard--dashboard-us-no-premium-scan-in-progress-resolved-breaches/).

Reply:
Renamed in [a930728](https://github.com/mozilla/blurts-server/pull/3640/commits/a93072894298a01f6d81d0aea77b3659b7a9d332).